content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,17 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked data from remote storage into the <abbr>cache</abbr> of the
+project (without placing it in the <abbr>workspace</abbr> like `dvc pull`). This
+makes them available for linking (or copying) into the workspace (refer to
+`dvc config cache.type`).
 
-Without arguments, `dvc fetch` ensures that the files specified in all
```

Comment:
I still don't see any significant improvements. We removed this paragraph and now we have to put `Note that this does not download [metafiles](/doc/user-guide/dvc-files-and-directories). That can be done with "git fetch" (if using Git).` note, which is very confusing the it is written now.

If you say something like `Note that this does not download [metafiles](/doc/user-guide/dvc-files-and-directories)` you expect user to think "Probably this command downloads metafiles" or asking that question. It's hard for me to imagine it happening that way. What unasked question we are trying to resolve with this note?

Reply:
OK. The "Without arguments" wasn't removed, just moved to after the diagram. The `git fetch` note is a different change (nor meant as a replacement).

Reply:
> What unasked question we are trying to resolve with this note?

At some point we decided to clarify that "DVC is not Git" in the analogous commands, which is why I added it. But now that it's here (even if I fixed it) I agree it seems unnecessary. At least it should be unnecessary... So I removed it for now.